conda/meta.yaml

```diff
@@ -0,0 +1,91 @@
+{% set major = load_file_regex(load_file='CMakeLists.txt', regex_pattern='(?<=set\(GADGETRON_VERSION_MAJOR )([^\)]+)') %}
+{% set minor = load_file_regex(load_file='CMakeLists.txt', regex_pattern='(?<=set\(GADGETRON_VERSION_MINOR )([^\)]+)') %}
+{% set patch = load_file_regex(load_file='CMakeLists.txt', regex_pattern='(?<=set\(GADGETRON_VERSION_PATCH )([^\)]+)') %}
+
+package:
+  name: gadgetron
+  version: {{ major.group(1) }}.{{ minor.group(1) }}.{{ patch.group(1) }}
+
+source:
+  path: ../
+
+requirements:
```

Comment:
Is there anyway to make this sync'ed with `environment.yml`. I'm a bit worried that we now have two places for this.

Reply:
It's certainly doable. 

I had the same thought and had a look at doing this with jinja templating (similar to the version number). Unfortunately the yaml python module used for parsing does not expose comments so the convention we use (# dev comment or similar) wouldn't work with that sort of implementation.

So we could generate meta.yaml within package.sh, but it wouldn't be as clean as using jinja support directly (homegrown script?). We would also have to enforce strong conventions in environment.yml (for signifying runtime vs build time dependencies) which has its own trade offs. 

Alternatively we could validate that the versions defined in meta.yml match those in environment.yml and avoid generation entirely.

Reply:
I think the last suggestion (validating versions) would be fine. So we would error if they are not in sync.